specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Subscriptions.json

```diff
@@ -444,6 +547,7 @@
             }
           }
         },
+        "isDataAction": false,
```

Comment:
`isDataAction` cannot set `false` here. It should annotated as an object with type.

Reply:
Hi @raych1, do you have an example of the object? I remember reading the readme rules help guide regarding this change and "false" is the example they gave as the correct value. I can't find it now since it was an error from last pipeline build.